manifests/m/Microsoft/WebDeploy/3.6/Microsoft.WebDeploy.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.WebDeploy
+PackageVersion: 3.6

```

Comment:
```suggestion
PackageVersion: 10.0.1973
```

Reply:
I am sorry, the version in the MSI package is incorrect - the real one is 3.6